osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -186,6 +185,48 @@ public string StatisticsJson
         [JsonProperty("position")]
         public int? Position { get; set; }
 
+        public IEnumerable<(HitResult result, int count, int? maxCount)> GetStatisticsForDisplay()
+        {
+            foreach (var key in OrderAttributeUtils.GetValuesInOrder<HitResult>())
+            {
+                if (key.IsBonus())
+                    continue;
+
+                int value = Statistics.GetOrDefault(key);
+
+                switch (key)
+                {
+                    case HitResult.SmallTickHit:
+                    {
+                        int total = value + Statistics.GetOrDefault(HitResult.SmallTickMiss);
```

Comment:
Yeeeeeah... I think this is best case and nothing more that can be done here... For lazer scores (timeshift) this'll work corectly since miss/hit directly correspond.

Good catch though, noted.